GoogleAuth.md

```
@@ -0,0 +1,41 @@
+Authentication (Google OAuth)
+
+This project uses Google OAuth 2.0 for user authentication via @react-oauth/google.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)